src/main/resources/i18n/Validation_de.properties

```diff
@@ -0,0 +1,102 @@
+#
+# This resource bundle is used for the query validation code to produce i18n messages
+#
+# The keys have the format of rule class name and then message type within that.  Most rules
+# will only have 1 or 2 message keys
+#
+# Please try and keep this sorted within rule class and use # between sections so the IDEA Ctrl-Alt-L reformat does not bunch
+# them too tightly.
+#
+# REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
+# so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
+#
+# Prior to Java 9, properties files are encoded in ISO-8859-1.
+# We have to use \u00fc instead of the German ue character, \u00e4 for ae, \u00f6 for oe, \u00df for ss
+#
+ExecutableDefinitions.notExecutableType=Validierungsfehler ({0}) : Die Typdefinition ''{1}'' ist nicht ausf\u00fchrbar
+ExecutableDefinitions.notExecutableSchema=Validierungsfehler ({0}) : Die Schemadefinition ist nicht ausf\u00fchrbar
+ExecutableDefinitions.notExecutableDirective=Validierungsfehler ({0}) : Die Direktivendefinition ''{1}'' ist nicht ausf\u00fchrbar
+ExecutableDefinitions.notExecutableDefinition=Validierungsfehler ({0}) : Die angegebene Definition ist nicht ausf\u00fchrbar
+#
+FieldsOnCorrectType.unknownField=Validierungsfehler ({0}) : Feld ''{1}'' vom Typ ''{2}'' ist nicht definiert
+#
+FragmentsOnCompositeType.invalidInlineTypeCondition=Validierungsfehler ({0}) : Die Bedingung f\u00fcr den Inline-Fragmenttyp ist ung\u00fcltig, muss auf Objekt/Schnittstelle/Union stehen
+FragmentsOnCompositeType.invalidFragmentTypeCondition=Validierungsfehler ({0}) : Die Bedingung des Fragmenttyps ist ung\u00fcltig, muss auf Objekt/Schnittstelle/Union stehen
+#
+KnownArgumentNames.unknownDirectiveArg=Validierungsfehler ({0}) : Unbekanntes Direktivenargument ''{1}''
+KnownArgumentNames.unknownFieldArg=Validierungsfehler ({0}) : Unbekanntes Feldargument ''{1}''
+#
+KnownDirectives.unknownDirective=Validierungsfehler ({0}) : Unbekannte Direktive ''{1}''
+KnownDirectives.directiveNotAllowed=Validierungsfehler ({0}) : Direktive ''{1}'' ist hier nicht erlaubt
+#
+KnownFragmentNames.undefinedFragment=Validierungsfehler ({0}) : Undefiniertes Fragment ''{1}''
+#
+KnownTypeNames.unknownType=Validierungsfehler ({0}) : Unbekannter Typ ''{1}''
+#
+LoneAnonymousOperation.withOthers=Validierungsfehler ({0}) : Anonyme Operation mit anderen Operationen
+LoneAnonymousOperation.namedOperation=Validierungsfehler ({0}) : Operation ''{1}'' folgt der anonymen Operation
+#
+NoFragmentCycles.cyclesNotAllowed=Validierungsfehler ({0}) : Fragmentzyklen nicht erlaubt
```

Comment:
Fragmentzyklen looks really odd for me ... If you want to translate it, I would probably make two words out of it.